src/index.js

```diff
@@ -22,17 +22,15 @@ const constants = {
 
 async function createCommand(ctx, CommandClass, args) {
   ctx.testCmd = new CommandClass(args, ctx.fakeConfig);
-  ctx.testCmd.secureStorage = {
-    async getCredentials(profileId) {
+  if (ctx.testCmd.userConfig) {
+    sinon.replace(ctx.testCmd.userConfig, 'getProfileById', profileId => {
```

Comment:
Need some context here, we have modified `getProfileById`'s return params, where are we using this ?  Below changes suggest we are modifying `ctx.userConfig.addProfile` also?

Reply:
Currently for each command that inherits [twilioClientCommand](https://github.com/twilio/twilio-cli-core/blob/main/src/base-commands/twilio-client-command.js#L33) we check the current profile before executing the command. Since previously we would use secureStorage (which uses keytar) to get credentials which is now removed. We needed to mock the response for getting current profile. hence we are mocking the getProfileById method

Reply:
Makes sense, thanks.